pkg/utils/utils.go

```diff
@@ -408,3 +411,31 @@ func IsValidHexValue(v string) bool {
 
 	return true
 }
+
+// GetLazydockerRootDirectory finds lazydocker root directory.
+//
+// It's used for our cheatsheet script and integration tests. Not to be confused with finding the
+// root directory of _any_ random repo.
+func GetLazydockerRootDirectory() string {
+	path, err := os.Getwd()
+	if err != nil {
+		panic(err)
+	}
+
+	for {
+		_, err := os.Stat(filepath.Join(path, ".git"))
+		if err == nil {
+			return path
+		}
+
+		if !os.IsNotExist(err) {
+			panic(err)
+		}
+
+		path = filepath.Dir(path)
+
+		if path == "/" {
+			log.Fatal("must run in lazydocker folder or child folder")
+		}
+	}
+}
```

Comment:
I copied over from `lazygit`, but I was wondering if a shared package would fit this case (Maybe `lazyutils`?). There seems to be quite a bit of overlap between them. Even these scripts for cheatsheets could be used in both projects. Let me know your thoughts 😄.

Reply:
@gusandrioli I actually made a repo for this exact purpose over the weekend: https://github.com/jesseduffield/lazycore. If you would like to raise a PR on that repo for shared cheatsheet functionality, I'll happily review :) Also happy to give pointers

Reply:
Ah neat! I'll open a PR for cheatsheet functionality there then 😄.